GithubSetup

```diff
@@ -0,0 +1 @@
+Subproject commit a73c2743725df2e1ec2f2327e330adecdb316e17
```

Comment:
I don't either, it was in the original pr. I'll take it out (I'm guessing it was a random file that got added in)